app/src/main/res/layout/view_item.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal"
+    android:padding="12dp">
```

Comment:
@ongebo please declare this in the dimens resource file

Reply:
Do you mean the padding @joshNic?

Reply:
I believe he's referring to the padding @ongebo